www/app/hardware/Hardware.js

```diff
@@ -4793,23 +4793,29 @@ define(['app'], function (app) {
 											});
 											PluginParams += '</select></td>';
 										} else {
-											var multiline=(typeof (param.multiline) != "undefined") && (param.multiline == "true")
+											var nbRows=-1;
+											if (typeof (param.rows) != "undefined") {
+												var n = Math.floor(Number(param.rows));
```

Comment:
How about just use

`nbRows = parseInt(param.rows)`

Reply:
Just to be sure to ignore wrong values like 10px or -10

Reply:
But it's maybe not the best way to do it.

Reply:
Then after the parseInt just check if it's smaller <1 or >10 and set a default number like 5 (or 1)

Reply:
I'm pretty sure parseInt("10e3") for example returns 10 for example. But it's probably not a big deal, I can do it if you want.

Reply:
That's fine as I wont expect any plugin author is going to enter this value... or any arbitrary value, even if he/she does, they would know when testing their plugin of course ;)

Reply:
Ok, it's done